src/04_printing.py

```diff
@@ -14,4 +14,5 @@
 
 # Use the 'format' string method to print the same thing
```

Comment:
99% of the time you'll be using f-strings, so I'm not really worried about you skipping over these, but there's two prompts here to print using the print operator `%` and the `format` method on strings. 

Also, note that `y` is rounded to precision 2 in the requested output: `y is 2.25`

If you feel up for the challenge, see if you can complete these two while making use of format codes for rounding: https://thepythonguru.com/python-string-formatting/

You can also use format codes in f-strings.